packages/js/src/index.ts

```diff
@@ -10,8 +10,12 @@ export * from './utils/typescript/ast-utils';
 export * from './utils/package-json';
 export * from './utils/assets';
 export * from './utils/package-json/update-package-json';
+export * from './utils/add-local-registry-scripts';
```

Comment:
This doesn't need to be public API, to be honest, this can live within `@nx/plugin` for now.

Reply:
removed